File: Zestawy Zadań/C++/5 - Funkcje, rekurencja/Zestaw 2 zadania/5.8 Funkcja o prototypie bool CZYBLISKIE(folat x, float y, float D).cpp
#include <iostream>
#include <cmath>

bool CZYBLISKIE(float x, float y, float D) {
    return std::fabs(x - y) < D;
}

int main() {
    float x, y;
    std::cout << "Podaj pierwsz¹ liczbê rzeczywist¹: ";
    std::cin >> x;
    std::cout << "Podaj drug¹ liczbê rzeczywist¹: ";
    std::cin >> y;
    if (CZYBLISKIE(x, y, 1)) {
        std::cout << "OK" << std::endl;
    } else {
        std::cout << "PECH" << std::endl;
    }
    return 0;
}

File: Zestawy Zadań/Python/5 - Funkcje, rekurencja/Zestaw 1 zadania/5.8 Funkcja o prototypie bool CZYBLISKIE(folat x, float y, float D).py
def CZYBLISKIE(x, y, D):
    return abs(x - y) < D

if __name__ == "__main__":
    x = float(input("Podaj pierwszą liczbę rzeczywistą: "))
    y = float(input("Podaj drugą liczbę rzeczywistą: "))
    if CZYBLISKIE(x, y, 1):
        print("OK")
    else:
        print("PECH")
